dev-docs/bidders/responsiveads.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: bidder
+title: ResponsiveAds
+description: ResponsiveAds Prebid Adapter
+biddercode: responsiveads
+media_types: banner
+pbjs: true
+pbs: false
+enable_download: true
+gvl_id: 1189
+sidebarType: 1
+safeframes_ok: false
+---
```

Comment:
Can you please add as much information as possible: https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter 

Especially:
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
deals_supported: true/false
floors_supported: true/false
fpd_supported: true/false

Reply:
Thank you @ChrisHuie.  I just added a commit with some more information.